lib/kernel/doc/kernel_app.md

```diff
@@ -130,6 +130,26 @@ For more information about configuration parameters, see file
     node. If a node goes down, it must thereafter be explicitly connected. See
     `m:net_kernel`.
 
+- **`epmd_module = module()`{: #epmd_module }** - Configures the module
+  responsible for communication with [epmd](epmd_cmd.md). If this parameter
```

Comment:
```suggestion
  responsible for communication with [epmd](`e:erts:erl_cmd.md`). If this parameter
```

Reply:
actually I did a copy paste error in my fix proposal above. It should of course link to `epmd_cmd.md` and not `erl_cmd.md`.

Reply:
And when checking I looked at the links below, so I didn't notice. Updated!